src/components/FeaturedPlantersSlider/index.js

```diff
@@ -152,8 +152,8 @@ function FeaturedPlantersSlider({
                   textAlign: 'center',
                   wordBreak: 'break-word',
                   display: '-webkit-box',
-                  '-webkit-line-clamp': '2',
-                  '-webkit-box-orient': 'vertical',
+                  WebkitLineClamp: '2',
+                  WebkitBoxOrient: 'vertical',
```

Comment:
shouldn't be webkitLineClamp, I feel camel case is the convention on the CSS name in react?

Reply:
@dadiorchen This is a special case. I checked Devtools and these css properties are applied. More info: https://github.com/emotion-js/emotion/issues/2036#issuecomment-705159259